### PR TITLE
chore: remove empty Component fields.

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: addon/ng2/blueprints/component/files/src/app/__path__/__name__/__name__.ts [hunk 1]
```diff
@@ -4,13 +4,8 @@ import {Component} from 'angular2/core';
 @Component({
   selector: '<%= dasherizedModuleName %>',
   templateUrl: 'app<%= dynamicPath %>/<%= dasherizedModuleName %>/<%= dasherizedModuleName %>.html',
-  styleUrls: ['app<%= dynamicPath %>/<%= dasherizedModuleName %>/<%= dasherizedModuleName %>.css'],
```
Comment (jkuri) [inline addon/ng2/blueprints/component/files/src/app/__path__/__name__/__name__.ts]: Why also styleUrls?

Comment (hansl) [inline addon/ng2/blueprints/component/files/src/app/__path__/__name__/__name__.ts]: Removed the `,`

**Comment by jkuri** [comment]
LGTM

**Comment by zackarychapple** [comment]
No merge?

**Comment by hansl** [comment]
No merge. Mike is changing the templates already.

**Pull request closed without merging**